test/common/schemaform/ObjectField.unit.spec.jsx

```diff
@@ -151,4 +151,52 @@ describe('Schemaform: ObjectField', () => {
     expect(tree.everySubTree('ExpandingGroup')).not.to.be.empty;
     expect(tree.subTree('ExpandingGroup').props.open).to.be.true;
   });
+  it('should handle change', () => {
+    const onChange = sinon.spy();
+    const onBlur = sinon.spy();
+    const schema = {
+      type: 'object',
+      properties: {
+        test: {
+          type: 'string'
+        }
+      }
+    };
+    const tree = SkinDeep.shallowRender(
+      <ObjectField
+          schema={schema}
+          idSchema={{}}
+          onChange={onChange}
+          onBlur={onBlur}/>
+    );
+
+    tree.getMountedInstance().onPropertyChange('test')('value');
+
+    expect(onChange.firstCall.args[0]).to.eql({
```

Comment:
That would work here as far as I know, but I did it this way because it's is a more precise test. The onChange function should be called with an object with just one property called `test`.